Api.php

```diff
@@ -261,4 +255,83 @@ public function getCaptureStrategy()
 
         return Constants::INTERFACE_TRANSACTION;
     }
+
+    protected function addOptionalInterfaceParams(string $interface, array $payload): array
+    {
+        $allowedOptions = [
+            Constants::INTERFACE_PAYMENT_PAGE => [
+                'config_set',
+                'payment_methods',
+                'wallets',
+                'notification_merchant_email',
+                'notification_payer_email',
+                'styling_css_url',
+                'styling_content_security_enabled',
+                'styling_theme',
+            ],
+            Constants::INTERFACE_TRANSACTION => [
+                'config_set',
+                'payment_methods',
+                'styling_css_url', // deprecated
+                'styling_content_security_enabled',
+                'styling_theme',
+            ]
+        ];
+
+        $optionalInterfaceOptions = $this->options['optionalParameters'] ?? [];
+
+        // legacy
+        if (null !== $this->options['iframeCssUrl']) {
+            $optionalInterfaceOptions['styling_css_url'] = $this->options['iframeCssUrl'];
+        }
```

Comment:
I think you can safely remove iframeCssUrl. [These clones](https://packagist.org/packages/karser/payum-saferpay/stats) are from Travis and my private CI/CD.